kaldb/src/main/java/com/slack/kaldb/logstore/opensearch/OpenSearchAdapter.java

```diff
@@ -196,7 +202,8 @@ public void reloadSchema() {
               entry.getValue().name);
         }
       } catch (Exception e) {
-        LOG.error("Error parsing schema mapping for {}", entry.getValue().toString(), e);
+        throw new IllegalStateException(
```

Comment:
I'm not sure this is what we want to do here? 

We need to provide protection against registering a field that already exists - this is a race condition that is expected to occur when new fields are being added.